pkgs/applications/misc/edgetx/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, mkDerivation, fetchFromGitHub
+, cmake, gcc-arm-embedded, python3Packages
+, qtbase, qtmultimedia, qttranslations, SDL, gtest
+, dfu-util, avrdude
+}:
+
+mkDerivation rec {
+  pname = "edgetx";
+  version = "2.5.0";
+
+  src = fetchFromGitHub {
+    owner = "EdgeTX";
+    repo = pname;
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-VcwlIgGl3PF3Sh9kkD6dUA/yMqidJRT9+8zGpkQucck=";
+  };
+
+  nativeBuildInputs = [ cmake gcc-arm-embedded python3Packages.pillow ];
+
+  buildInputs = [ qtbase qtmultimedia qttranslations SDL ];
+
+  postPatch = ''
+    sed -i companion/src/burnconfigdialog.cpp \
```

Comment:
Could this be using `substituteInPlace`?

Reply:
Turns out, [avrdude was removed completely](https://github.com/EdgeTX/edgetx/commit/5d4f6679e5d4715223f7a8e8982d824de7c12594). And yes, I think I can use substituteInplace here

Reply:
However, I'm not in favor of using substituteInplace. [There](https://github.com/EdgeTX/edgetx/blob/main/companion/src/burnconfigdialog.cpp#L88) is both `/usr/bin/local/dfu-util` and `/usr/bin/dfu-util`. Catching both cases using the regex seems like an elegant solution to me?

Reply:
Ah, that's fair. IMO `substituteInPlace` still looks a bit cleaner, but you make a good point.